### PR TITLE
Video Background

### PR DESCRIPTION
**Pull request by raywan**
I got video working. It's currently just a sample video. There's no text overlay, let's discuss what we should put later.

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -58,80 +58,10 @@
         <header class="masthead">
             <div class="slider-container" id="slider">
```
Comment (ntimpham) [inline app/index.html]: can also remove everything here. no need for any slider/tp stuff

```
<div class="slider-container" id="slider">
    <div class="tp-banner-container">
```

File: app/index.html [hunk 2]
```diff
@@ -58,80 +58,10 @@
...
-                                </div>
-                                <div class="tp-caption sfb hidden-xs" data-x="520" data-y="center" data-hoffset="0" data-voffset="150" data-speed="1000" data-start="1700" data-easing="Power4.easeOut">
-                                    <a href="#about" class="btn btn-primary btn-lg">LEARN MORE</a>
-                                </div>
-                            </li>
-
-                        </ul>
-                        <div class="tp-bannertimer"></div>
-                    </div>
+                    <video autoplay style="width:100%">
```
Comment (ntimpham) [inline app/index.html]: can we add a way to pause?
`onclick="this.paused ? this.play() : this.pause();"` will do for now I think.

Comment (raywan) [inline app/index.html]: Sure I can do that. In production though, the video just be autoplaying in the background right?

**Comment by raywan** [comment]
@ntimpham Made the changes, but some shit happened with git lol.

**Comment by ntimpham** [comment]
Lol did you merge master into your branch? Anyways good to merge 👍 

As for prod, if the video is just those audio-less looping ones, yea no need to pause.

**Comment by clemmy** [comment]
interesting how you got the PR to look like this lel

**Pull request merged**